scripts/portfolio.js

```diff
@@ -0,0 +1,28 @@
+(function(){
```

Comment:
You dont need to put 'use strict' inside of an iife.  It works without the iife.